Service/src/util/db_connection.py

```diff
@@ -2,26 +2,19 @@
 import os
 import simplejson as json
 
-CONFIG_FILE_PATH = 'Service/db/config.json'
-ROOT_FOLDER_NAME = 'stockdog-api/'
-
-def getConfigFilePath():
-   cwd = os.getcwd()
-   strIdx = cwd.find(ROOT_FOLDER_NAME)
-   return cwd[:strIdx + len(ROOT_FOLDER_NAME)] + CONFIG_FILE_PATH
-
+CONFIG_FILE_PATH = './config.json'
```

Comment:
Yea man Idk if I'm a fan of relative paths like that, I put in all that logic to do a proper path for a reason before. Not a blocker and it's still _ok_ but just wanted to voice that out.

Reply:
I see what you mean, if we end up moving the config file it would entail that we need to update all of the usages of this file. I think a good solution is creating a utility function for accessing config variables. I'll see about making the that function and import it using absolute pathing.